pkg/util/util.go

```diff
@@ -54,6 +54,34 @@ func IsGPUVMI(vmi *v1.VirtualMachineInstance) bool {
 	return false
 }
 
+// IsGPUVMIDevicePlugins checks if a VMI has any GPUs configured for device plugins
```

Comment:
I think utils is a bad choice for all of these functions. There should be either host devices common library or something more specific than utils.

Reply:
the functions are used in virt-controller, when the manifest is rendered with device-plugin gpu or dra GPU. For now I can move it to virt-controller